content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,241 @@
+gets a short-lived, automatically rotating token using the `TokenRequest` API
+and mounts the token as a
+[projected volume](/docs/concepts/storage/projected-volumes/#serviceaccounttoken).
+If the Pod or the ServiceAccount is deleted, the token is no longer valid in the
+Kubernetes API.
+
+<!-- OK to remove this historical detail after Kubernetes 1.31 is released -->
+
+In versions earlier than 1.22, Kubernetes provides a long-lived, static token
+to the Pod as a Secret.
+
+#### Manually retrieve ServiceAccount credentials {#get-a-token}
+
+If you need the credentials for a ServiceAccount to mount in a non-standard
+location, or for an audience that isn't the API server, use one of the
+following methods:
+
+* [TokenRequest API](/docs/reference/kubernetes-api/authentication-resources/token-request-v1/)
+  (recommended): Request a short-lived service account token using your
+  *application code*. The token expires automatically and can rotate upon
+  expiration.
+* [Token Volume Projection](/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
+  (recommended): In Kubernetes v1.20 and later, use the Pod specification to
+  tell the kubelet to add the service account token to the Pod as a
+  *projected volume*. Projected tokens expire automatically, and the kubelet
+  rotates the token before it expires. The ServiceAccount token is also
+  invalidated in the Kubernetes API when the Pod or the ServiceAccount is
+  deleted.
+* [Service Account Token Secrets](/docs/tasks/configure-pod-container/configure-service-account/#manually-create-a-service-account-api-token)
+  (not recommended): You can mount service account tokens as Kubernetes
+  Secrets in Pods. These tokens don't expire and don't rotate. This method
+  is not recommended, especially at scale, because of the risks associated
+  with static, long-lived credentials. In Kubernetes v1.24 and later, the
+  [LegacyServiceAccountTokenNoAutoGeneration feature gate](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features)
+  prevents Kubernetes from automatically creating these tokens for
+  ServiceAccounts.
+
+## Authenticating service account credentials {#authenticating-credentials}
+
+ServiceAccounts use signed
+{{<glossary_tooltip term_id="jwt" text="JSON Web Tokens">}}  (JWTs)
+to authenticate to the Kubernetes API server, and to any other system where a
+trust relationship exists. Depending on the token issuer, a ServiceAccount
+token might also have an expiry time, an audience, and a time after which the
+token *starts* being valid. When a process running in a Pod attempts to
+communicate with the Kubernetes API server, it adds an
+`Authorization: Bearer <token>` header to the HTTP request. The API server
+checks the validity of the bearer token as follows:
+
+1. Check the token signature.
+1. Check whether the token has expired.
+1. Check whether the token is currently valid.
```

Comment:
This is rather vague, all the other points are a also a check that the token is currently valid. Did you mean any specific check?

Tokens can  be (and very often are) bound to a certain API object. We may want to mention that check here, too.

Reply:
What are the checks that are actually appropriate @stlaz? That information will help @shannonxtreme tweak the doc.

Reply:
I see only Pod and Secret are currenly allowed to be referenced.

The code for this specific authentication lives in https://github.com/kubernetes/kubernetes/blob/8a72f059997c548c5432980682f50ac7bfa036ae/pkg/serviceaccount/jwt.go#L262

along with the validator in https://github.com/kubernetes/kubernetes/blob/8a72f059997c548c5432980682f50ac7bfa036ae/pkg/serviceaccount/claims.go#L111

Reply:
I would hesitate to enumerate all the "token is currently valid" checks the API server does here in a way that encourages other parties to replicate those checks outside the API server, but something like "Checks whether object references in the token claims are still valid" could work

Reply:
Added this suggestion